pkgs/applications/networking/cluster/k3s/default.nix

```diff
@@ -0,0 +1,233 @@
+with import <nixpkgs> {};
```

Comment:
this line fails to build on hydra: https://hydra.nixos.org/build/116132269

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/84369

Reply:
I wonder how this wasn't caught on ofborg. cc @grahamc